SeriousGame/Assets/Custom/Scripts/points.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class points : MonoBehaviour {
```

Comment:
Class names should begin with a capital letter

Reply:
@larsheesakkers1